Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -1041,3 +1171,37 @@ void MenuBar::PatchHLEFunctions()
 {
   HLE::PatchFunctions();
 }
+
+void MenuBar::ClearCache()
+{
+  Core::RunAsCPUThread(JitInterface::ClearCache);
+}
+
+void MenuBar::LogInstructions()
+{
+  PPCTables::LogCompiledInstructions();
+}
+
+void MenuBar::SearchInstruction()
+{
+  bool good;
+  QString op = QInputDialog::getText(this, tr("Change instruction"), tr("New instruction:"),
+                                     QLineEdit::Normal, QStringLiteral(""), &good);
+
+  if (!good)
+    return;
+
+  bool found = false;
+  for (u32 addr = 0x80000000; addr < 0x80180000; addr += 4)
```

Comment:
`0x80180000` should be `0x81800000`.

Reply:
That just means its already wrong in Wx :P

https://github.com/dolphin-emu/dolphin/blob/a9987588eb4fbd69a6fb9f405a83827084b1a874/Source/Core/DolphinWX/Debugger/CodeWindow.cpp#L553